numpy/testing/tests/test_utils.py

```diff
@@ -432,6 +437,109 @@ def test_nan_items(self):
                 lambda: self._assert_func(ainf, anan))
 
 
+class TestArrayAssertLess(unittest.TestCase):
+
+    def setUp(self):
+        self._assert_func = assert_array_less
```

Comment:
Really, release notes, and maybe double check if the tests cover all important and it should be good.